airbyte-cdk/java/airbyte-cdk/core/src/test/kotlin/io/airbyte/cdk/integrations/util/ConnectorExceptionHandlerTest.kt

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+package io.airbyte.cdk.integrations.util
+
+import io.airbyte.commons.exceptions.ConfigErrorException
+import io.airbyte.commons.exceptions.ConnectionErrorException
+import io.airbyte.commons.exceptions.TransientErrorException
+import java.sql.SQLException
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+internal class ConnectorExceptionHandlerTest {
+
+    // Concrete subclass for testing
+    private class TestConnectorExceptionHandler : ConnectorExceptionHandler() {}
```

Comment:
you could follow this with `val exceptionHandler = TestConnectorExceptionHandler()` and reduce clutter

Reply:
actually I don't need this anymore.. `ConnectorExceptionHandler` used to be an abstract class, which has to be extended. Now it is a regular class, so I can just have a member variable initialized:

```
val exceptionHandler = ConnectorExceptionHandler();

```

Reply:
fixed.